modules/siddhi-core/src/main/java/org/wso2/siddhi/core/aggregation/IncrementalExecutor.java

```diff
@@ -235,7 +235,14 @@ private void dispatchEvent(long startTimeOfNewAggregates, BaseIncrementalValueSt
                 LOG.debug("Event dispatched by " + this.duration + " incremental executor: " + eventChunk.toString());
             }
             if (isProcessingExecutor) {
-                executorService.execute(() -> table.addEvents(tableEventChunk, 1));
+                executorService.execute(() -> {
+                    try {
+                        table.addEvents(tableEventChunk, 1);
+                    } catch (Throwable t) {
+                        LOG.error("Exception occurred when writing to aggregation table of duration '" +
```

Comment:
Please add siddhiApp name and aggregation name too

Reply:
Ack.. Will work on improving the parser to pass that information to the executor level